src/core/bittorrent/session.cpp

```diff
@@ -2128,13 +2131,16 @@ void Session::handleTorrentRemovedAlert(libtorrent::torrent_removed_alert *p)
 
 void Session::handleTorrentDeletedAlert(libt::torrent_deleted_alert *p)
 {
+    m_savePathsToRemove.remove(p->info_hash);
+}
+
+void Session::handleTorrentDeleteFailedAlert(libt::torrent_delete_failed_alert *p)
+{
+    // libtorrent won't delete the directory if it contains files not listed in the torrent,
+    // so we remove the directory ourself
```

Comment:
"// so we remove the directory ourself" -> "// so we remove the directory **ourselves**"